homeassistant/components/alarm_control_panel/mqtt.py

```diff
@@ -166,3 +166,7 @@ def alarm_arm_away(self, code=None):
                          self._payload_arm_away, self._qos)
         else:
             _LOGGER.warning("Wrong code entered while arming away!")
+
+    def alarm_trigger(self, code=None):
```

Comment:
had a pylint error and decided to add them to mqtt and verisure... I'll remove them and add a pylint exception in both